resources/tests.mjs

```diff
@@ -391,6 +391,30 @@ Suites.push({
     ],
 });
 
+Suites.push({
+    name: "Charts-chartjs",
+    url: "tentative/charts/dist/chartjs.html",
+    async prepare(page) {},
+    tests: [
+        new BenchmarkTestStep("Prepare", (page) => {
+            page.querySelector("#prepare").click();
+        }),
+        new BenchmarkTestStep("Draw scatter", (page) => {
+            page.querySelector("#add-scatter-chart-button").click();
+        }),
+        // Note: this step needs the asynchronous runner to show up in results properly,
```

Comment:
This note is not necessary because we're working around that now, right?

Reply:
ah good catch!